core/new-gui/src/app/common/service/notification/notification.service.ts

```diff
@@ -24,26 +25,27 @@ export class NotificationService {
     this.notificationStream.next(notification);
   }
 
-  success(message: string) {
-    this.sendNotification({ type: "success", message });
+  success(message: string, duration: number = 3000) {
```

Comment:
instead of passing only duration, since the notification service is a wrapper for `NzMessageService`, can we wrap the API so that it takes any option of NzMessageService API? see details here https://ng.ant.design/components/message/en#api

Reply:
changed.